src/Maths/math.isovector.ts

```diff
@@ -0,0 +1,289 @@
+import { Logger } from "../Misc/logger";
+import { Vector3} from "./math.vector";
+import { THIRDR3, IsoGridSize} from './math.constants';
+import { DeepImmutable} from "../types";
+import { ArrayTools } from '../Misc/arrayTools';
+import { _TypeStore } from '../Misc/typeStore';
+
+/**
+ * Class representing an isovector containing 2 INTEGER coordinates
+ * x axis is horizontal
+ * y axis is 60 deg counter clockwise from positive y axis

```

Comment:
I am also wondering if it should only extend Vector2 as a lot of code is dupplicated and only a couple specific rotation and such have been added ?

Reply:
_Would be could if you have a reference like wiki_ As this is something I made up to expedite the project I doubt if you will find a corresponding definition anywhere.

 _if it should only extend Vector2_ Problem here is that other Vector2 properties do not produce anything sensible. 

Since the class is so specialized and only used for the purpose of creating the Geodesic and Goldberg polyhedra it probably makes more sense to remove unused properties and hide the class from the API doc?

Reply:
Sounds perfect to me if you hide it !!!

Reply:
Unused properties removed, class hidden to API docs - done